core/src/main/java/com/linecorp/armeria/common/AggregatedHttpMessage.java

```diff
@@ -20,12 +20,15 @@
 
 import javax.annotation.Nullable;
 
+import com.linecorp.armeria.common.util.UnstableApi;
+
 /**
  * A complete HTTP message whose content is readily available as a single {@link HttpData}. It can be an
  * HTTP request or an HTTP response depending on what header values it contains. For example, having a
  * {@link HttpHeaderNames#STATUS} header could mean it is an HTTP response.
  */
-interface AggregatedHttpMessage {
+@UnstableApi
```

Comment:
Perhaps OK to remove this?

Reply:
Yeah intended to mark that users shouldn't need to implement this but guess it doesn't hurt either